content/api/hub/sample_ollama.md

```diff
@@ -0,0 +1,87 @@
+title = "Rust HTTP trigger template"
```

Comment:
```suggestion
title = "OLlama using Spin"
```
Can we please use this new name, to prevent clashes with existing items in the hub?
Thanks so much.

Reply:
Renders correctly if using unique name:
![Screenshot 2023-12-13 at 10 32 51](https://github.com/fermyon/developer/assets/9831342/7f9c6595-cbb0-4041-a0cd-01ec21189922)